includes/components/class-attribute.php

```diff
@@ -734,6 +734,26 @@ public function add_field_settings( $meta_box ) {
 						'html'       => true,
 						'_order'     => 120,
 					];
+
+					if('attachment_upload' === $field_type){
```

Comment:
Instead of excluding a single type let's add a list of supported types instead, as a condition (using in_array), similarly to the other task "require unique values..." (I added a comment there). This way we ensure that custom field types created by other devs will also not be affected by this.

Reply:
As far as I've seen, the problem only appears with checkboxes